rsocket-core/src/main/java/io/rsocket/loadbalance/LoadbalanceTarget.java

```diff
@@ -0,0 +1,76 @@
+/*
+ * Copyright 2002-2020 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.rsocket.loadbalance;
+
+import io.rsocket.transport.ClientTransport;
+
+/**
+ * Simple container for a key and a {@link ClientTransport}, representing a specific target for
+ * loadbalancing purposes. The key is used to compare previous and new targets when refreshing the
+ * list of target to use. The transport is used to connect to the target.
+ *
+ * @since 1.1
+ */
+public class LoadbalanceTarget {
+
+  final String key;
+  final ClientTransport transport;
+
+  private LoadbalanceTarget(String key, ClientTransport transport) {
```

Comment:
The routing broker won't have a `ClientTransport` but an `RSocket`.

Reply:
IIRC the broker only depends on `LoadBalanceStrategy` or do I have that wrong?

Reply:
There hasn't been integration yet. So maybe that's all we'll use just some discussion about. If that is the case, then this won't be a problem. Maybe @OlegDokuka remembers. I can definitly see using this on the routing client side though.

Reply:
We do not need transport for broker. Only  strategy

Reply:
thanks for the remindert